### ISSUE TITLE
iframe onLoad jsx-a11y/no-noninteractive-element-interactions error

### ISSUE THREAD

**Issue by claramunt** [jsx-eslint/eslint-plugin-jsx-a11y#474]
Im getting error on iframes when setting onLoad eventhandler.
It was fixed in https://github.com/evcohen/eslint-plugin-jsx-a11y/pull/281 but with last version its failing again

**Comment by ljharb** [comment]
Since we still have test cases, can you provide the code it’s warning on?

**Comment by claramunt** [comment]
I think the test about iframe onLoad event is no longer.
The code that is warning on is this 
```
              <iframe
                    name="embeddedExternalPayment"
                    ref="embeddedExternalPayment"
                    style={iframeStyle}
                    onLoad={this.handleLoadIframe}
               />
```

**Comment by ljharb** [comment]
That code passes when I add a test case for it.

Can you provide the full text of the warning, and the full code it's warning on?

**Comment by ljharb** [comment]
what is “[eslint]” from? Where are you getting this warning, and can you try on the command line?

It’s possible that being inside a Fragment affects it but I’m not sure how it could - if you change the fragment to a div, what happens?

**Comment by claramunt** [comment]
I copy/pasted from visual studio, but from command line I get the same. 
```
158:17  error  Non-interactive elements should not be assigned mouse or keyboard event listeners  jsx-a11y/no-noninteractive-element-interactions
```
When I change the fragment to a div, I still get the same error

**Comment by ljharb** [comment]
did you figure it out?

**Comment by evcohen** [comment]
@claramunt has this been resolved?

**Comment by claramunt** [comment]
> @claramunt has this been resolved?

Still getting the same error

**Comment by evcohen** [comment]
what version are you using?

**Comment by claramunt** [comment]
I was using 6.1.1, but also tried with 6.1.2

**Comment by chrisworfolksky** [comment]
+1. I'm using `6.2.0` and I'm getting warnings for img tags with an onLoad.

    <img
      ref={this.ref}
      className="c-responsive-image-placeholder__image"
      src={src}
      alt={alt}
      data-test-id="test-id"
      onLoad={this.fetchCompleteImage}
    />

**Comment by thedaviddias** [comment]
Is this still a bug? `onLoad` seems valid on img and iframes but the "no-noninteractive-element-interactions" rule says it's not. Any insights?

**Comment by V2dha** [comment]
I think this is resolved as all the code above are passing with `img` and `iframe` with `onLoad` in version `6.6.0`
